### DISCUSSION TITLE
Support for svelte

### DISCUSSION THREAD

**Discussion by akkumar** [mdx-js/mdx#1853] [General] [answered]
This is regarding mdx v2 ( rc status etc. ) . 

mdxjs seems to support esm modules with v2.  will it support svelte out of the box or is there any other plugin needed to make it work ? 

there is another project called mdsvex that seems to do something like this  but it would be great if it is natively supported here .

Is there a documentation that I am missing to indicate svelte is supported. 

Would appreciate links  / pointers if any.

**Comment by wooorm** [comment, marked as answer]
https://mdxjs.com/docs/getting-started/#svelte.
(I just merged this an hour ago btw. This should work. Would appreciate if you could test it out though)
